app/views/moderator_applications/show.html.slim

```diff
@@ -0,0 +1,38 @@
+.container-fluid
+  .row
+    .col-lg-2
+
+    main.restore-defaults.col-lg-8
+      h1
+        = link_to(@application.user.username, user_path(@application.user))
+        | 's Moderator Application
+
+      - if current_user.is_admin?
```

Comment:
I shouldn't be using this `is_admin?` check everywhere; it's non-semantic and inflexible. There should be a separate predicate along the lines of `can_view_moderator_applications?` or something.

Reply:
The style guide recommends avoiding prefixes like `can_`, but I'm not sure what the alternative would be (`views_moderator_applications?`?)

Reply:
i'll look into this myself